theme.css

```diff
@@ -211,6 +211,42 @@ form.message-search input {
 	color: var(--body-color);
 }
 
+/* Nick Colour Fix for 4.4.0 */
```

Comment:
Get rid of that comment, it isn't useful in the future for readers of the code.
They will be working with the latest version.

Reply:
Comment removed.